### PR TITLE
Removed note that Composer install setup not done

### PR DESCRIPTION
**Pull request by asgrim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Takes the XML output from [Psalm](https://psalm.dev/) and renders it as HTML.
 
 First, install `xsltproc` on your machine (for example, `apt install xsltproc`).
 
-Then `composer require --dev roave/paslm-html-output` (note: not yet set up)
+Then `composer require --dev roave/paslm-html-output`
```
Comment (Ocramius) [inline README.md]: `paslm`?

Comment (Ocramius) [inline README.md]:
```suggestion
Then `composer require --dev roave/paslm-html-output`
Then `composer require --dev roave/psalm-html-output`
```

Comment (Ocramius) [inline README.md]:
```suggestion
Then `composer require --dev roave/psalm-html-output`
```

Comment (asgrim) [reply README.md]: Fixed

**Pull request merged**